packages/fast-components-styles-msft/src/utilities/color/neutral-divider.ts

```diff
@@ -0,0 +1,22 @@
+import { DesignSystem, DesignSystemResolver } from "../../design-system";
+import { neutralDividerDelta } from "../design-system";
+import {
+    findClosestBackgroundIndex,
+    getSwatch,
+    palette,
+    Palette,
+    PaletteType,
+} from "./palette";
+
+export const neutralDivider: DesignSystemResolver<string> = (
+    designSystem: DesignSystem
+): string => {
+    const neutralPalette: Palette = palette(PaletteType.neutral)(designSystem);
+    const backgroundIndex: number = findClosestBackgroundIndex(designSystem);
+    const delta: number = neutralDividerDelta(designSystem);
+    const swapThreshold: number = neutralPalette.length - delta;
+    const direction: 1 | -1 = backgroundIndex >= swapThreshold ? -1 : 1;
+
+    const index: number = backgroundIndex + direction * delta;
+    return getSwatch(index, neutralPalette);
```

Comment:
I think this should export the product of the product of the `colorRecipeFactory` so the function signature is consistent with the other color recipes.

Also, should this be namespaced to `rest` in case we add states to this?

Reply:
+1 on the namespace to `rest`. I can actually think of a scenario where we have a divider that also functions as a drag handle where we may want hover and focus states.